### DISCUSSION TITLE
How should resolved issues be handled / when should resolved issues be closed?

### DISCUSSION THREAD

**Discussion by gnalvesteffer** [Teardown-Issue-Tracker-Maintainers/Teardown-Issue-Tracker#34] [General]
Some proposals I've been suggested:
1. Confirm that the issue is actually resolved, and then close the issue. If a dev says that an issue will be fixed in a future update, then a maintainer or dev would confirm this on release and close the issue.
2. Close the issue upon marking it as resolved, and if the issue still persists post-update, re-open the issue.

Opinions? I personally like the 2nd proposal since it reduces the amount of open issues.

Vote for the preferred proposal by this Sunday: https://strawpoll.com/ab5zsoaq6

**Comment by kmcgurty** [comment]
Option 2 is much less cumbersome. Anyone is able to comment on a closed issue - if the issue isn't actually fixed, it can be re-opened until it's fixed again. As you said, it also cleans up the open issues.

**Comment by Thomasims** [comment]
My reasoning for supporting option 1 was purely visibility. Someone who finds an issue isn't going to look in closed issues as those are supposed to be dealt with and no longer relevant. That isn't the case if players can still stumble upon the issue in-game.

I do agree that this can create some clutter in the issues list, but that argument only applies to those who don't use filters; You can very easily filter out dev/resolved issues with `-label:dev/resolved `. As a maintainer looking for issues that need to be updated you should already be using filters anyway and as a user looking for the issue you're experiencing the state of that issue shouldn't matter for your search.

It all just comes down to what these 2 options stand for and what direction we want to take. Option 1 is, in my opinion, the more functional one as it allows for the extra state of "This issue is solved in the next update" over Option 2 which is just cleaning up for the sake of cleaning up.

**Reply by kmcgurty** [reply]
> Someone who finds an issue isn't going to look in closed issues

Anyone worth their salt will, I know I certainly have for unrelated repos. Besides, duplicate issues are inevitable, they aren't really that big of a deal.

Also, it's not just for cleanliness - having to test every single "fixed" issue after every update would be extremely tedious. I certainly don't mind helping the community but that's just doing extra work for no reason.

**Reply by Thomasims** [reply]
You missed my point, closed issues are done and dealt with. Of course some people are going to look in closed issues as well but they shouldn't have to. If an issue is still active within the game it should still be reflected on the issue tracker, marked as resolved or not.

We don't necessarily need to test every issue before closing as I'm sure that's done internally anyway but it's good to double check. If you don't want to confirm a fix yourself then leave it to somebody else or tag the original poster asking if they consider the problem resolved. Nobody is asking you to do extra work for no reason.

**Comment by gnalvesteffer** [comment]
I’ve created a poll. Vote on a proposal by this Sunday. Whichever has the majority vote will take effect next week: https://strawpoll.com/ab5zsoaq6

**Comment by gnalvesteffer** [comment]
The results are tied, but TuxedoLabs has been closing the tickets lately, so I think we should just let the devs close resolved tickets.
